src/main/java/graphql/validation/rules/SubscriptionRootField.java

```diff
@@ -39,16 +47,24 @@ public void checkOperationDefinition(OperationDefinition operationDef) {
 
             GraphQLObjectType subscriptionType = getValidationContext().getSchema().getSubscriptionType();
 
+            // This coercion takes into account default values for variables
```

Comment:
This is fixing an edge case - this check previously didn't take into account default variable values because there was previously no coercion.

That caused my tests to fail (see later) because I had set a default boolean value for include and skip.